ngraph/cmake/external_protobuf.cmake

```diff
@@ -97,7 +97,7 @@ else()
             CXX_VISIBILITY_PRESET default
             C_VISIBILITY_PRESET default
             VISIBILITY_INLINES_HIDDEN OFF)
-        set_target_properties(libprotobuf libprotobuf-lite PROPERTIES
+        set_target_properties(${_proto_libs} libprotobuf-lite PROPERTIES
```

Comment:
It seems to be equivalent of my PR: https://github.com/openvinotoolkit/openvino/pull/4948/files
I think that your version can work not correctly for `protobuf-lite` case. In such a case we have:
`set_target_properties(libprotoc libprotobuf-lite libprotobuf-lite ...`
and the flags will be not applied to `libprotobuf` which is built always (even in `protobuf-lite` scenario)